.github/workflows/test-graphql.yml

```
@@ -0,0 +1,29 @@
+name: Test GraphQL generation
```

my suggestion might be a bit long, idk

```suggestion
name: Ensure GraphQL schema and generated code is up-to-date
```